### PR TITLE
MAWB-1: Configure telegram client

### PR DESCRIPTION
**Pull request by Illia1927**

### CODE REVIEW

File: src/main/java/mate/academy/bot/Bot.java [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+            execute(sendMessage);
+        } catch (TelegramApiException e) {
+            e.printStackTrace();
+        }
+    }
+
+    @Override
+    public String getBotUsername() {
+        return "wiki_mate_academy_bot";
+    }
```
Comment (YatKat) [inline src/main/java/mate/academy/bot/Bot.java]: hard code (use properties)

File: src/main/java/mate/academy/bot/Bot.java [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+import org.telegram.telegrambots.ApiContextInitializer;
+import org.telegram.telegrambots.bots.TelegramLongPollingBot;
+import org.telegram.telegrambots.meta.TelegramBotsApi;
+import org.telegram.telegrambots.meta.api.methods.send.SendMessage;
+import org.telegram.telegrambots.meta.api.objects.Update;
+import org.telegram.telegrambots.meta.exceptions.TelegramApiException;
+
+public class Bot extends TelegramLongPollingBot {
+    public static void main(String[] args) {
```
Comment (boroda4436) [inline src/main/java/mate/academy/bot/Bot.java]: @Illia1927 We don't need the main method here
Please, create BotConfig and register bot there. Annotate BotConfig as a @ Component

File: src/main/java/mate/academy/bot/Bot.java [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+        try {
+            execute(sendMessage);
+        } catch (TelegramApiException e) {
+            e.printStackTrace();
+        }
+    }
+
+    @Override
+    public String getBotUsername() {
+        return "wiki_mate_academy_bot";
```
Comment (boroda4436) [inline src/main/java/mate/academy/bot/Bot.java]: Please, fetch this from application properties. Use @ Value annotation

**Comment by boroda4436** [comment]
@Illia1927 Any updates on this ticket?

**Comment by Dinexpod** [comment]
Did you solve all issues in check-style? You don't have a green tick...

**Pull request closed without merging**